src/model/trust.model.ts

```diff
@@ -28,20 +29,22 @@ export interface Trust {
   creation_date_month: string;
   creation_date_year: string;
   unable_to_obtain_all_trust_info: string;
-  INDIVIDUALS?: TrustIndividual[];
+  INDIVIDUALS?: (TrustIndividual | GeneralTrustee) [];
   HISTORICAL_BO?: TrustHistoricalBeneficialOwner[];
   CORPORATES?: TrustCorporate[];
 }
 
 interface TrustIndividual {
-  type: string;
+  id?: string;
+  type: BeneficialOwnerTypeChoice | RoleWithinTrustType;
   forename: string;
   other_forenames: string;
   surname: string;
   date_of_birth_day: string;
   date_of_birth_month: string;
   date_of_birth_year: string;
   nationality: string;
+  second_nationality?: string;
```

Comment:
This will also need to be added in the overseas-entities-api and chips-filing-consumer. Add a note to the Jira and tag Leah and Sarah so future stories can be added

Reply:
Also this will need to be added to the `api-sdk-node` otherwise the `overseas-entities-api` won't get it. As well asking Leah and Sarah for a new Jira I think we need the Test story for this slice to emphasis for the testers that these are new optional fields and they need to do focused testing on it